src/main/resources/db/migration/V5.0.0.1__7177-newmetrics.sql

```diff
@@ -0,0 +1,13 @@
+
+alter table metric add column if not exists dataverse_id bigint;
+
+DO $$
+BEGIN
+
+  BEGIN
+    alter table metric add constraint fk_metric_dataverse_id FOREIGN KEY (dataverse_id) REFERENCES dataverse(id);
+  EXCEPTION
+    WHEN duplicate_object THEN RAISE NOTICE 'Table constraint foo.bar already exists';
```

Comment:
I assume this isn't supposed to say "foo.bar" but I doubt it's a big deal.

Reply:
:-) - caught at cut/paste !